src/sentry/grouping/strategies/newstyle.py

```diff
@@ -541,10 +541,7 @@ def single_exception(
     rv = {}
 
     for variant, stacktrace_component in stacktrace_variants.items():
-        values = [
-            stacktrace_component,
-            system_type_component if variant == "system" else type_component,
-        ]
```

Comment:
Why/how can the system component go away?

Reply:
Good catch. I will fix it.